docs/pages/tutorials/embedded-mode-with-sqlite-nextjs.mdx

```diff
@@ -0,0 +1,277 @@
+
+import Link from 'next/link';
+import { lists } from '.keystone/api';
+
+export default function Home({ post }) {
+  return (
+    <div>
+      <main style={{margin: "3rem"}}>
+      <div>
+        <Link href="/">
+          <a>&larr; back home</a>
+        </Link>
+      </div>
+      <h1>{post.title}</h1>
+      <p>{post.content}</p>
+      </main>
+    </div>
+  );
+}
+
+export async function getStaticPaths() {
+  const posts = await lists.Post.findMany({
+    query: `slug`,
+  });
+
+  const paths = posts
+    .map(post => post.slug)
+    .filter((slug): slug is string => !!slug)
+    .map(slug => `/post/${slug}`);
+
+  return {
+    paths,
+    fallback: false,
+  };
+}
+
+export async function getStaticProps({ params: { slug } }) {
+  const [post] = await lists.Post.findMany({
+    where: { slug },
+    query: 'id title content',
+  });
+  return { props: { post } };
+}
+```
+
+Run `yarn dev` again.
+
+**Congratulations!** 🙌 &nbsp; You now have:
+
+- A Next.js frontend blending static pages from your frontend repo
+- Dynamic pages powered by Keystone content that‘s editable in an intuitive Admin UI.
+
+![Navigating between the home page and post pages](/assets/tutorials/embedded-nextjs/localhost-home-to-post.gif)
+
+## Bonus: add the GraphQL API to the frontend
+
+To get a read-only GraphQL API and playground in production, rename `/pages/api/hello.tsx` to `graphl.tsx` with the following:
+
+```tsx
+// pages/api/graphql.tsx
+
+export { default, config } from '.keystone/next/graphql-api';
+```
+
+This takes the fully functional GraphQL API that Keystone is already generating and makes it available as an endpoint and playground within the Next.js frontend app at [http://localhost:3000/api/graphql](http://localhost:3000/api/graphql).
+
+![A browser displaying the GraphQL playground](/assets/tutorials/embedded-nextjs/graphql-api.png)
+
+This gives you the ability to implement a search against the same content API you have running at build time, in run time.
+
+## Bonus: deploy to Vercel
+
+To get your project on the internet via Vercel hosting complete the following steps:
+
+1. Commit your project to a repository in [Github](https://github.com/), [Gitlab](https://gitlab.com/), or [Bitbucket](https://bitbucket.org/).
+2. Login to [Vercel](https://vercel.com/) using an account from the services above.
+3. Create a new Vercel project and link it to your newly created repository.
+
+![A browser displaying the GraphQL playground](/assets/tutorials/embedded-nextjs/vercel-new-project.gif)
+
+## Summary
+
+Keystone’s introduction of SQLite gives you the option to run a self contained CMS from the same place you keep your frontend code. While this option restricts read-write access to people who can run the project in local development, it has advantages with ease of setup, security, and web deployment. [Embedded mode](/concepts/modes#embedded-mode) is also a great way to deploy a read-only API on the web for content you manage on your computer.
```

Comment:
Using the term `introduction` pins this content to a point in time. Every day that passes the "introduction" becomes less relevant that its "support".

```suggestion
Keystone’s support of SQLite gives you the option to run a self contained CMS from the same place you keep your frontend code. While this option restricts write access to people who can run the project in local development, it has advantages with ease of setup, security, and web deployment. [Embedded mode](/concepts/modes#embedded-mode) is also a great way to deploy a read-only API on the web for content you manage on your computer.
```